### ISSUE TITLE
Gem doesn't work with Sprockets 3 beta 6

### ISSUE THREAD

**Issue by waynehoover** [maxd/less-rails-semantic_ui#2]
Hi, if you try and run this gem with sprockets 3.0.0.beta.6 it gives this error:

`undefined method '[]' for nil:NilClass`

I'm not sure if this is a less-rails issue or its from this gem. I would expect though its a less-rails issue.

**Comment by maxd** [comment]
Could you please provide simple Rails project for reproduce this problem?

**Comment by waynehoover** [comment]
Yes, here is a sample project:

https://github.com/waynehoover/semantic-less-sprockets-3

I have just installed less-rails-semantic_ui into a new Rails 4.2.0 project, I'm using ruby 2.2.0. run `bundle` then `rails s` and go to the root url to see the error. Then downgrade sprockets version 2.x and then see that it runs fine.

I did a little debugging and it looks like the issue is with less_rails like I expected, so not sure this is the correct place for this bug.
